packages/cli/src/util/get-project-name.ts

```diff
@@ -2,7 +2,7 @@ import { basename } from 'path';
 import { VercelConfig } from '@vercel/client';
 
 export interface GetProjectNameOptions {
-  argv: { '--name'?: string };
+  argv: { [k in string]?: string };
```

Comment:
this doesn't seem right. does it need to be changed? did you mean this?

```suggestion
  argv: { [k: string]: string };
```

Reply:
yeah i think so. typescript didn't seem to mind though 😆

Reply:
<img width="498" alt="Screenshot 2023-06-28 at 15 00 51" src="https://github.com/vercel/vercel/assets/16144158/65f5cd99-ffc8-438c-8c0d-7056abeed3b1">
This is what im getting locally. :thinkies:

Reply:
did we need to change this line though? what's the purpose?

Reply:
Now that the input to `args()` is generated from the data structure, it messed with some of the types. I don't really understand it, but this was the only spot typescript was failing. This change fixes it

Reply:
The `argOptions` on line 102 i think trickles down to this type change. Idk why. Theres some TypeScript magic at work in the `args` package

Reply:
`{ [k: string]: string | undefined }`

Reply:
fixed